examples/paymentbot.py

```diff
@@ -55,7 +57,8 @@ def start_without_shipping_callback(bot, update):
     description = "Payment Example using python-telegram-bot"
     # select a payload just for you to recognize its the donation from your bot
     payload = "Custom-Payload"
-    # get your provider_token at @botfather, see https://core.telegram.org/bots/payments#getting-a-token
+    # In order to get a provider_token to get your provider_token see
```

Comment:
In order to get a provider_token ~~to get your provider_token~~ see https://core.telegram.org/bots/payments#getting-a-token